server/models/flavor.js

```diff
@@ -7,14 +7,28 @@ const flavorSchema = mongoose.Schema({
 
 const FlavorModel = mongoose.model('Flavor', flavorSchema);
 
+saveFlavor = function(flavorData) {
```

Comment:
For consistency we might wanna stick with function name(input) {} rather than name = function(input){}